### DISCUSSION TITLE
Windows scroll bar configure to be always visible and always expanded...

### DISCUSSION THREAD

**Discussion by jhastingsiii** [dotnet/maui#20178] [Ideas]
### Description

A big usability issue I run into with users of my app is that the Windows scroll bar is not very noticeable. I've done certain things to help mitigate this problem like adding various borders around scrollable content, but the scroll bar is still just too small. It's an acceptable size, albeit still a bit small, when you mouse over it though and I'd like to be able to configure it to where it stays that size at all times.

Example of not moused over:
![image](https://github.com/dotnet/maui/assets/13514539/17343dae-ddc7-4d5c-8a21-342345fa6178)

Example of moused over:
![image](https://github.com/dotnet/maui/assets/13514539/ec308ae7-d936-4784-b0d0-b4d577b9c0ec)

You can see in the first picture the scrollbar is barely noticeable at all, I'd like to be able to make it always show as it is in the second picture above. In an ideal world I'd like it to look/behave like the scroll bar in the Edge browser, the Edge browser scroll bar is twice as thick.

I've read over all the documentation on a ScrollView and can't seem to find any relevant settings to do this. I'd welcome any suggestions for accomplishing this or workarounds if they are possible.

### Public API Changes

I'm not sure what would be good for an API change for this feature...

Maybe an attribute on any components with a scrollbar like ScrollBarBehavior="AlwaysExpanded" or something?

### Intended Use-Case

Makes scrollbars more visible.

**Comment by nvrnight** [comment]
Are there any plans for improving the customization of the scrollbar?
